src/sagas/account.js

```diff
@@ -27,31 +27,32 @@ import { getTokenName } from '../selectors/env';
 import { minDelayCall } from './helpers';
 
 export function * doSignup (action) {
-  const { formData } = action;
+  const { formData, nextRoute } = action;
+
   try {
     const response = yield minDelayCall(api.signup, formData);
 
     // Persist token for future page loads
     const tokenName = yield select(getTokenName);
-    new Cookies().set(tokenName, response.key, { path: '/' });
```

Comment:
I believe we still need to set the path here, I am not sure what the path defaults too if the path is not set but setting it to `'/'` ensures that it is accessible to all pages

Reply:
I thought it was using `path` to redirect to the home page.  I can change it back.